test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js

```diff
@@ -3,9 +3,7 @@ define(['common/cellComponents/tabs/infoTab', 'common/props', 'testUtil'], (
     Props,
     TestUtil
 ) => {
-    'use strict';
-
-    describe('The App Info Tab module', () => {
+    fdescribe('The App Info Tab module', () => {
```

Comment:
need to remove the `f` so that the whole test suite runs

Reply:
oops!